ror-tools/src/test/scala/tech/beshu/ror/tools/RorToolsAppSuite.scala

```diff
@@ -132,6 +208,178 @@ class RorToolsAppSuite extends AnyWordSpec with BeforeAndAfterAll with BeforeAnd
           |  -h, --help               prints this usage text""".stripMargin
       )
     }
+    "Patching not started because there is a metadata file indicating that the ES is already patched" in {
+      File(backupDirectory.path).createDirectory()
+      patchMetadataFile.createFile()
+      patchMetadataFile.write(
```

Comment:
IMO, we have too many details at this level. Reading the test, the reader wants to know what is the given state. They are not interested on how the state was achieved. So, maybe we should have a private method like this:
```
givenPatchMetadata(rorVersion = "0.0.1", esVersionUsed, patchedFilesMetadata = List.empty)
```

All the details should be in the method